frontend/src/pages/MilestonesEditor.tsx

```diff
@@ -515,15 +516,13 @@ export const MilestonesEditor = () => {
                       </div>
                       <div className={classes(css.summaryWrapper)}>
                         <MilestoneRatingsSummary
-                          tasks={versionLists[version.id] || []}
+                          tasks={list}
                           completed={completed}
                         />
                       </div>
-                      {!completed && !!versionLists[version.id].length && (
```

Comment:
I'd still leave the length check in, since we decided not to show MilestoneCompletedness on empty milestones.
![image](https://user-images.githubusercontent.com/80758782/168037274-f005c07f-720d-4f92-96ef-574e658761da.png)

Reply:
Yeah, missed this at first
Added the length check back in